source/patterns/02-organisms/content-header/content-header.twig

```diff
@@ -16,6 +16,15 @@
       {%- endblock categories -%}
     {%- endwith -%}
 
+    <a href="" class="content-header__download_link">
+      <picture>
```

Comment:
We should go SVG-only for MVP. We can always generate other formats as / when we need to.